python/cuml/tests/test_cuml_descr_decor.py

```diff
@@ -38,6 +37,10 @@
     np.float64,
 ]
 
+unsupported_cudf_dtypes = [
+    np.uint8, np.uint16, np.uint32, np.uint64, np.float16
+]
+
```

Comment:
Why not just import them like before?

Reply:
We no longer define these in the library itself because our definition fell out of date with cuDF's implementation and artificially constrained what we would and wouldn't accept. Now we just enumerate them for testing purposes, but these tests should probably be refactored later to avoid this as well.